third_party/move/move-compiler-v2/src/file_format_generator/module_generator.rs

```diff
@@ -372,15 +372,15 @@ impl ModuleGenerator {
         let type_parameters = fun_env
             .get_type_parameters()
             .into_iter()
-            .map(|TypeParameter(_, TypeParameterKind { abilities, .. })| abilities)
+            .map(|TypeParameter(_, TypeParameterKind { abilities, .. }, _loc)| abilities)
```

Comment:
```suggestion
            .map(|TypeParameter(_, TypeParameterKind { abilities, .. }, _)| abilities)
```

Reply:
nit: it is a bit weird to name one of your ignores, but not the other.

Reply:
Well, everyone knows the first field is always `NodeId`.  But what is the 3d one?

Reply:
Well, "everyone" is incorrect: the first field here is symbol, not node id.